src/UWP/Package.appxmanifest

```diff
@@ -3,8 +3,8 @@
   <Identity Name="fb84989a-4cf4-440f-a76d-ef3506d9c9ba" Publisher="CN=Hicham" Version="1.0.0.0" />
   <mp:PhoneIdentity PhoneProductId="fb84989a-4cf4-440f-a76d-ef3506d9c9ba" PhonePublisherId="00000000-0000-0000-0000-000000000000" />
   <Properties>
-    <DisplayName>UWP</DisplayName>
-    <PublisherDisplayName>Hicham</PublisherDisplayName>
+    <DisplayName>Bitwarden</DisplayName>
+    <PublisherDisplayName>Alistair</PublisherDisplayName>
```

Comment:
Please change this to `8bit Solutions LLC`, which is what the app would be published under.